In-house patch to remove confusing namespace warning message in the log file.

*** neutron-2015.1.2/neutron/agent/common/config.py	Tue Oct 13 10:35:16 2015
--- new/neutron/agent/common/config.py	Fri Jun  3 13:25:51 2016
***************
*** 54,61 ****
  USE_NAMESPACES_OPTS = [
      cfg.BoolOpt('use_namespaces', default=True,
                  help=_("Allow overlapping IP. This option is deprecated and "
!                        "will be removed in a future release."),
!                 deprecated_for_removal=True),
  ]
  
  IPTABLES_OPTS = [
--- 54,60 ----
  USE_NAMESPACES_OPTS = [
      cfg.BoolOpt('use_namespaces', default=True,
                  help=_("Allow overlapping IP. This option is deprecated and "
!                        "will be removed in a future release.")),
  ]
  
  IPTABLES_OPTS = [
